app/backend/src/couchers/servicers/search.py

```diff
@@ -55,6 +58,11 @@
 TRI_SIMILARITY_THRESHOLD = 0.6
 TRI_SIMILARITY_WEIGHT = 5
 
+conversational_fluent_filter_values = [
+    fluency2sql[api_pb2.LanguageAbility.Fluency.FLUENCY_CONVERSATIONAL],
+    fluency2sql[api_pb2.LanguageAbility.Fluency.FLUENCY_FLUENT],
```

Comment:
No need to go through `fluency2sql` here, you can just have `LanguageFluency.conversational` and `LanguageFluency.fluent` from `models.py` (already imported).

Reply:
Actually it seems this is unused now?

I think it's better the frontend deals with what fluency level should be matched in the backend.

Reply:
i think its better to implement this is backend rather than someone explaining everything again to front end. IN that case should i create more like sql objects based version of conversational_fluent_filter_values ?